tests/system-tests/rdscore/internal/rdscoreconfig/config.go

```diff
@@ -502,9 +494,40 @@ type CoreConfig struct {
 	//nolint:lll
 	PodLevelBondPodSubnetMaskIPv6 string `yaml:"rdscore_pod_level_bond_pod_subnet_mask_ipv6" envconfig:"ECO_RDSCORE_POD_LEVEL_BOND_POD_SUBNET_MASK_IPV6"`
 	//nolint:lll
-	PodLevelBondPodMacAddr string `yaml:"rdscore_pod_level_bond_pod_mac_addr" envconfig:"ECO_RDSCORE_POD_LEVEL_BOND_POD_MAC_ADDR"`
-	WorkerLabelListOption  metav1.ListOptions
+	PodLevelBondPodMacAddr string         `yaml:"rdscore_pod_level_bond_pod_mac_addr" envconfig:"ECO_RDSCORE_POD_LEVEL_BOND_POD_MAC_ADDR"`
 	FRRExpectedNodes       EnvSliceString `yaml:"rdscore_frr_expected_nodes" envconfig:"ECO_RDSCORE_FRR_EXPECTED_NODES"`
+	RootlessDPDKNamespace  string         `yaml:"rdscore_rootless_dpdk_ns" envconfig:"ECO_RDSCORE_ROOTLESS_DPDK_NS"`
+	//nolint:lll,nolintlint
+	RootlessDPDKClientDeploymentName string `yaml:"rdscore_rootless_dpdk_client_deployment_name" envconfig:"ECO_RDSCORE_ROOTLESS_DPDK_CLIENT_DEPLOYMENT_NAME"`
+	//nolint:lll,nolintlint
+	RootlessDPDKNetworkOne string `yaml:"rdscore_rootless_dpdk_network_one" envconfig:"ECO_RDSCORE_ROOTLESS_DPDK_NETWORK_ONE"`
+	//nolint:lll,nolintlint
+	RootlessDPDKNetworkTwo string `yaml:"rdscore_rootless_dpdk_network_two" envconfig:"ECO_RDSCORE_ROOTLESS_DPDK_NETWORK_TWO"`
+	//nolint:lll,nolintlint
+	RootlessDPDKVlanID string `yaml:"rdscore_rootless_dpdk_vlan_id" envconfig:"ECO_RDSCORE_ROOTLESS_DPDK_VLAN_ID"`
+	//nolint:lll,nolintlint
+	RootlessDPDKDummyVlanID string `yaml:"rdscore_rootless_dpdk_dummy_vlan_id" envconfig:"ECO_RDSCORE_ROOTLESS_DPDK_DUMMY_VLAN_ID"`
+	//nolint:lll,nolintlint
+	RootlessDPDKNodeOne string `yaml:"rdscore_rootless_dpdk_node_one" envconfig:"ECO_RDSCORE_ROOTLESS_DPDK_NODE_ONE"`
+	//nolint:lll,nolintlint
+	RootlessDPDKNodeTwo string `yaml:"rdscore_rootless_dpdk_node_two" envconfig:"ECO_RDSCORE_ROOTLESS_DPDK_NODE_TWO"`
+	//nolint:lll,nolintlint
+	RootlessDPDKDeploymentSA string `yaml:"rdscore_rootless_dpdk_deployment_sa" envconfig:"ECO_RDSCORE_ROOTLESS_DPDK_DEPLOYMENT_SA"`
+	//nolint:lll,nolintlint
+	RootlessDPDKPolicyTwo string `yaml:"rdscore_rootless_dpdk_policy_two" envconfig:"ECO_RDSCORE_ROOTLESS_DPDK_POLICY_TWO"`
+	//nolint:lll,nolintlint
+	RootlessDPDKClientVlanMac string `yaml:"rdscore_rootless_dpdk_client_vlan_mac" envconfig:"ECO_RDSCORE_ROOTLESS_DPDK_CLIENT_VLAN_MAC"`
+	//nolint:lll,nolintlint
+	RootlessDPDKClientMacVlanMac string `yaml:"rdscore_rootless_dpdk_client_macvlan_mac" envconfig:"ECO_RDSCORE_ROOTLESS_DPDK_CLIENT_MACVLAN_MAC"`
+	//nolint:lll,nolintlint
+	RootlessDPDKClientIPVlanMac string `yaml:"rdscore_rootless_dpdk_client_ipvlan_mac" envconfig:"ECO_RDSCORE_ROOTLESS_DPDK_CLIENT_IPVLAN_MAC"`
+	//nolint:lll,nolintlint
+	RootlessDPDKClientIPVlanIPv4 string `yaml:"rdscore_rootless_dpdk_client_ipvlan_ipv4" envconfig:"ECO_RDSCORE_ROOTLESS_DPDK_CLIENT_IPVLAN_IPV4"`
```

Comment:
don't we need IPv6 for the parity?

Reply:
We can extend the test coverage with the ipv6 ip addr family of cause.
But there are a few questions:

- ipv6 for the sriov interface only, for the interface behind the TAP or both

- as far as I understand the feature, the ip family is not so important in this case

Please let me know WDYT; I will extend the TP coverage if needed

Reply:
OK, I'm leaving this up to U 😸 If there's no explicit request for IPv6 then let start with IPv4 and we could always extend the coverage later, thanks

Reply:
thank you! I'll see what can be done to enhance it a little bit later; probably it will be a separate MR if you'll agree. please let me know WDYT